spark-gremlin/pom.xml

```diff
@@ -227,6 +259,16 @@
             <artifactId>snappy-java</artifactId>
             <version>1.1.1.7</version>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.0.29.Final</version>
```

Comment:
That's an old version of netty and conflicts with what we currently use:

https://github.com/apache/tinkerpop/blob/b8a2452deb7f6a23fae29684c1ead2ed35d7cb69/gremlin-driver/pom.xml#L29

in fact, on master we've already bumped further ahead to:

https://github.com/apache/tinkerpop/blob/b8a2452deb7f6a23fae29684c1ead2ed35d7cb69/gremlin-driver/pom.xml#L29

Not sure what kind of conflicts this would create when spark is used as a plugin with the console/server. Any reason we can't make this coincide with what's current?